test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -850,6 +852,22 @@ private static Settings.Builder getExcludeSettings(int num, Settings.Builder bui
         return builder;
     }
 
+    /**
+     * Returns a list of the data stream's backing index names.
+     */
+    public List<String> getDataStreamBackingIndexNames(String dataStreamName) {
+        GetDataStreamAction.Response response = safeGet(
+            client().execute(
+                GetDataStreamAction.INSTANCE,
+                new GetDataStreamAction.Request(TimeValue.THIRTY_SECONDS, new String[] { dataStreamName })
```

Comment:
Nit:
```suggestion
                new GetDataStreamAction.Request(TEST_REQUEST_TIMEOUT, new String[] { dataStreamName })
```

Reply:
Fixed but offline to not initiate too many ci builds.